client/src/pages/home/home.js

```diff
@@ -1,52 +1,50 @@
-import React,{useState} from 'react';
```

Comment:
Looks like all of the quotes were changed to double form single. We should keep them single.

Reply:
The double quotes are from Material-UI, I have another branch after this one is merged with prettier and eslint implemented so after this is merged I will merge with that one and lint all the files and create a new PR

Reply:
Master branch didn't have double-quotes. Just trying to figure out where they came from. Did you completely redo home?